parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -89,7 +89,7 @@ UBUNTU_OS_NAME="UBUNTU"
 MARINER_OS_NAME="MARINER"
 KUBECTL=/usr/local/bin/kubectl
 DOCKER=/usr/bin/docker
-export GPU_DV=470.57.02
+export GPU_DV="{{GPUDriverVersion}}"
```

Comment:
this is no longer user for vhd build, but will dynamically select the correct version at runtime. for the NCv1 sizes which no longer support 510 series, this will mean egress calls.

Reply:
this also means the vanilla aks vhd will work for all GPU sizes however, which is the point.